akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -945,23 +974,28 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
         }
       case LoadAllCompleted ⇒
         log.debug(
-          "Loading {} entries from durable store took {} ms",
-          count, TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startTime))
+          "Loading {} entries from durable store took {} ms, stashed {}",
+          count, TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startTime), stash.size)
         context.become(normalReceive)
+        unstashAll()
         self ! FlushChanges
 
       case GetReplicaCount ⇒
         // 0 until durable data has been loaded, used by test
-        sender() ! ReplicaCount(0)
+        replyTo ! ReplicaCount(0)
 
       case RemovedNodePruningTick | FlushChanges | GossipTick ⇒
       // ignore scheduled ticks when loading durable data
       case m @ (_: Read | _: Write | _: Status | _: Gossip) ⇒
         // ignore gossip and replication when loading durable data
         log.debug("ignoring message [{}] when loading durable data", m.getClass.getName)
+      case msg: ClusterDomainEvent ⇒ normalReceive.applyOrElse(msg, unhandled)
+      case msg ⇒
+        stash :+= (msg → replyTo)
```

Comment:
I don't think that is needed. It's just in the init phase. What would it do after reaching the limit? E.g. a local `Get` is guaranteed to reply.